### PR TITLE
2.1.11

### PR DESCRIPTION
**Pull request by mjadach-iv**
Chagelog:
- add jazzicons to the navbar and node selector (jazzicon based off the node api endpoint)
- add node name to the navbar
- move current ticket price to balances on info subpage
- add tickets calculation to the configuration subpage

## Release Notes

- **New Features**
	- Enhanced `ConnectNode` component with dynamic node address icons and improved display of node information.
	- Introduced `showJazzIcon` property in the `Select` component for visual indicators alongside selection options.
	- Added ticket price calculations to the `SettingsPage` component, improving strategy data representation.

- **Bug Fixes**
	- Adjusted fallback logic for block numbers and checksums in the `InfoPage` component, improving data accuracy.

- **Documentation**
	- Updated `README.md` to remove outdated build instructions for the Staking Hub. 

- **Style**
	- New CSS styles added for node icons and select menu items to enhance visual presentation.

### CODE REVIEW

File: src/pages/node/configuration.tsx [hunk 1]
```diff
@@ -32,6 +33,113 @@ function SettingsPage() {
...
+                `"${key}": "${strategies.strategies.AutoRedeeming.on_close_redeem_single_tickets_value_min}", // tickets: ${ticketsString}`,
+              );
+            } else {
+              tmp = tmp.replace(
+                stringToReplace,
+                `"${key}": "${strategies.strategies.AutoRedeeming.on_close_redeem_single_tickets_value_min}" // tickets: ${ticketsString}`,
+              );
+            }
+          }
+        }
```
[the remark this answers is not in the document]
Comment (mjadach-iv) [reply src/pages/node/configuration.tsx:134]: No need as all the calculations are in the try/catch for that reason

File: .prettierignore [hunk 2]
```diff
@@ -6,4 +6,6 @@ public/
 src/index.tsx
 
 # Created by github pipeline
-gha-creds-*.json
\ No newline at end of file
+gha-creds-*.json
+
+src/utils/functions.ts
```
Comment (esterlus) [inline .prettierignore:11]: why exclude code from prettier?

Comment (mjadach-iv) [reply .prettierignore]: Fixed

File: src/pages/node/configuration.tsx [hunk 3]
```diff
@@ -32,6 +33,112 @@ function SettingsPage() {
...
+      } catch (e) {
+        console.warn('Error while counting strategies against current ticket price.', e);
+      }
+
+      set_strategiesString(tmp);
+    }
+  }, [strategies, ticketPrice]);
```
[the remark this answers is not in the document]
Comment (mjadach-iv) [reply src/pages/node/configuration.tsx:140]: will do in the next PR

**Review by esterlus — APPROVED** [review]
Nice work

**Pull request merged**